source/develop/release-management/features/network/provider-physical-network.md

```diff
@@ -50,6 +42,10 @@ There are several ways to use this feature.
 
 ![add an external network connected to a physical network ManageIQ dialog](/images/features/network/provider-physical-network_new-network-dialog-miq.png)
 
+- **Select physical network via a custom physical network name and optional VLAN.** This option is mostly for backward compatibility and for external providers that are not integrated with oVirt Data Center Networks as described in the first option. When creating a new external network, set `Physical Network` to physical network name, it will be passed to provider as `provider:physical_network`. If the physical network has VLAN tagging enabled, check `Enable VLAN tagging` and set the physical network's tag there, it will be passed to provider as `provider:segmentation_id`. When VLAN is specified, `provider:network_type` is set to `vlan`, `flat` otherwise. In case a network name is longer than 15 characters or contains special characters, this option would not work with the OVN implementation. Using the first option is advised.
+
```

Comment:
would -> will

implementation. Using the first 
->
implementation, and using the first